modules/ROOT/pages/introduction/use-exchange.adoc

```diff
@@ -0,0 +1,186 @@
+= Use Exchange to Discover Connectors, Templates, and Examples
+
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+Use Anypoint Exchange (Exchange) to discover connectors, templates, and examples, and to incorporate them into your app. Exchange serves as a repository for connectors, templates, and examples provided both by MuleSoft and by other organizations. Searches in Exchange apply only to the Exchange organization you are viewing.
```

Comment:
Use Anypoint Exchange to discover integration assets like Connectors, Templates & Examples and import them into your project/app. Exchange is a platform of discover, where API designers, application developers and architects are able to catalog, discover and reuse APIs and integration assets.

You can browse & search for assets in your organization's own Exchange, or you can also browse assets built & curated by MuleSoft in MuleSoft's Exchange, which is publicly available at https://www.mulesoft.com/exchange/, or in your own organization's Exchange, in the left navigation panel from the link: "Provided by MuleSoft"

@wwarner-salesforce : FYI